src/docfx/restore/RestoreWorkTree.cs

```diff
@@ -71,7 +71,7 @@ async Task AddWorkTrees()
                 await ParallelUtility.ForEach(hrefs, async href =>
                 {
                     var (_, rev) = GitUtility.GetGitRemoteInfo(href);
-                    var workTreeHead = await GitUtility.Revision(restorePath, rev);
+                    var workTreeHead = $"{await GitUtility.Revision(restorePath, rev)}-{rev}";
```

Comment:
Will this fail when `rev` contains InvalidFileChars, or contains `/` that makes path contains sub folder?